src/main/java/com/ecwid/maleorang/method/v3_0/lists/abuse_reports/AbuseReportInfo.kt

```diff
@@ -0,0 +1,91 @@
+package com.ecwid.maleorang.method.v3_0.lists
+
+import com.ecwid.maleorang.MailchimpObject
+import com.ecwid.maleorang.annotation.Field
+
+/**
+ * Created by apocheau on 27/08/16.
+ */
+class   AbuseReportInfo : MailchimpObject() {
+
+    @JvmField
+    @Field
+    var id: String? = null
+
+    @JvmField
+    @Field
+    var campaign_id: String? = null
+
+    @JvmField
+    @Field
+    var list_id: String? = null
+
+    @JvmField
+    @Field
+    var email_id: String? = null
+
+    @JvmField
+    @Field
+    var email_address: String? = null
+
+    @JvmField
+    @Field
+    var merge_fields: Object? = null
```

Comment:
This field should be of type MailchimpObject (so that the info could be taken from [mapping](http://static.javadoc.io/com.ecwid/maleorang/3.0-0.9.4/com/ecwid/maleorang/MailchimpObject.html#mapping)).